pulp_ansible/app/serializers.py

```diff
@@ -134,6 +134,12 @@ class CollectionRemoteSerializer(RemoteSerializer):
         max_length=2000,
     )
 
+    sync_dependencies = serializers.BooleanField(
+        help_text=_("Sync dependencies for collections specified in requirements file"),
+        allow_null=True,
+        default=False,
+    )
```

Comment:
we support mirroring and sync from requirements file, I believe when syncing from requirements it would always sync dependencies